index.js

```diff
@@ -45,6 +46,15 @@ app.get("*", (req, res) => {
   res.sendFile(path.join(__dirname, "build", "index.html"));
 });
 
+// ---------- IN-PROCESS WORKERS -------------
+
+const airtableInterval = parseInt(process.env.AIRTABLE_SYNC || 0, 10);
```

Comment:
The good eye with the rename.

The second param is the radix for parseInt, an eslint suggestion.
It used to be that if the user wrote "0100" without the radix it would get parsed in octal rather than decimal because javascript.
But, they fixed it in ES5 apparently! So I'll remove this so the code scans better and turn off the eslint rule in the config